test/build.js

```diff
@@ -118,4 +118,18 @@ describe('build', function () {
 
     emitter.emit('error', new Error('failed'));
   });
+
+  it('return clear error message when Meteor not installed', function (done) {
+    var message = [
+      'Meteor not in path.',
+      'Please make sure Meteor is installed properly.'
+    ].join(' ');
+
+    Build({}, function (err) {
+      expect(err.message).to.equal(message);
```

Comment:
To allow for subtle changes to the message without needing updated tests, you could try something like:

`expect(err.message).to.include('Meteor not in $PATH')`

_also, I think it should be `$PATH` not `path`_